models/marts/core/int_all_graph_resources.sql

```diff
@@ -34,6 +34,7 @@ unioned_with_calc as (
         *,
         case 
             when resource_type = 'source' then  {{ dbt.concat(['source_name',"'.'",'name']) }}
+            when version is not null then {{ dbt.concat(['name',"'.v'",'version']) }} 
```

Comment:
Why this vs. a new field called "version"?

Reply:
i add that on line 87!

Reply:
ope - yes, thank you!